guides/common/modules/proc_executing-a-remote-job.adoc

```diff
@@ -40,7 +40,7 @@ You can also define a one-time future job, or set up a recurring job.
 For recurring tasks, you can define start and end dates, number and frequency of runs.
 You can also use cron syntax to define repetition.
 ifndef::orcharhino[]
-For more information about cron, see the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/ch-automating_system_tasks[Automating System Tasks] section of the {RHEL} 7 _System Administrator's Guide_.
+For more information about cron, see https://www.redhat.com/sysadmin/automate-linux-tasks-cron[Automate your Linux system tasks with cron] in _Red Hat Enable Sysadmin_.
```

Comment:
The chapter available in RHEL 7 Docs was removed in RHEL 8 Docs.
I tried to find a replacement, this is the best I've got. Let me know if you can find a better one.

Reply:
We also have subscriber-exclusive solution: https://access.redhat.com/solutions/9247

But I would rather consider removing this link altogether.